ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  openstack-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-installer
+  openstack-tests:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-tests
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.14"
+  latest:
+    candidate:
+      product: ocp
+      stream: ci
```

Comment:
shouldn't it be the nightly stream as well?

Looking at https://github.com/openshift/release/blob/master/ci-operator/config/openshift/release/openshift-release-master__nightly-4.14.yaml#L86-L97, they do nightly -> nightly upgrade.